### DISCUSSION TITLE
Autodoc fails for API scripts

### DISCUSSION THREAD

**Discussion by elliotfontaine** [TaloDev/godot#50] [General]
I just wanted to confirm this isn't only happening to me before opening a issue.

API classes don't appear when searching for "API" or "Talo". But since each of them is registered as a property of Talo, it's still possible to access their autodoc through Talo's one.
<img width="722" alt="Capture d’écran 2024-12-07 à 00 43 17" src="https://github.com/user-attachments/assets/188701e1-08b8-45f6-9e71-f256fde85002">

Accessing through Talo properties 👇
<img width="667" alt="Capture d’écran 2024-12-07 à 00 50 13" src="https://github.com/user-attachments/assets/6552c8ff-6385-434f-a429-8c088bc4dd4e">
<img width="727" alt="Capture d’écran 2024-12-07 à 00 50 35" src="https://github.com/user-attachments/assets/d0c85743-7fbf-4949-aa37-db41dc8e6013">

There are a lot of autodocs issues right now (e.g. https://github.com/godotengine/godot/issues/72406), many of which will be fixed in 4.4 I believe.

**Comment by tudddorrr** [comment]
Yeah this happens for me too. A few times (very rarely) when adding the docs I did manage to get them to show up automatically but lets wait and see if 4.4 fixes this.
